hippylib/test/test_gaussian_real_prior.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (c) 2016-2018, The University of Texas at Austin 
+# & University of California, Merced.
+#
+# All Rights reserved.
+# See file COPYRIGHT for details.
+#
+# This file is part of the hIPPYlib library. For more information and source code
+# availability see https://hippylib.github.io.
+#
+# hIPPYlib is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License (as published by the Free
+# Software Foundation) version 2.0 dated June 1991.
+
+from __future__ import absolute_import, division, print_function
+
+import unittest 
+import dolfin as dl
+import numpy as np
+import scipy.stats as scistat
+
+import sys
+sys.path.append('../../')
+from hippylib import *
+
+def get_prior_sample(pr):
+
+    noise = dl.Vector()
+    pr.init_vector(noise, "noise")
+    parRandom.normal(1.0, noise)
+    
+    s = dl.Vector()
+    pr.init_vector(s, 0)
+    pr.sample(noise, s, add_mean=True)
+    
+    return s
+
+class TestGaussianRealPrior(unittest.TestCase):
+    """
+    Test suite for prior.GaussianRealPrior
+    """
+
+    def setUp(self):
```

Comment:
I am not so familiar with style guides in unit tests, but is it necessary to have so much white space?

Reply:
@eldarkh ,

I don't think there's any specific style about it.
We could leave it as is, but if you'd like to remove some empty lines is OK too.